doc/sphinx/yaml/species.rst

```diff
@@ -305,6 +458,23 @@ Example::
       molar-volume: 1.3 cm^3/mol
 
 
+.. _sec-yaml-coverage-dependent-surface-species:
+
+Coverage-dependent Surface
+--------------------------
+
+A model where species thermodynamic properties are calculated as a function
+coverage as :ct:`described here <CoverageDependentSurfPhase>`.
+
+Additional fields:
+
+``coverage-dependencies``
+    Mapping where keys are the name of species whose coverage affects
+    thermodynamic properties of the node-owner species. The map values are
+    the dependency entries including ``model`` and model-specific parameters as
+    described in :ref:`sec-yaml-species-coverage`.
+
+
```

Comment:
This placement still implies that the ``coverage-dependencies`` field is a child of the ``equation-of-state`` field, but it's actually a top-level field within each species entry, i.e. alongside `thermo` and `equation-of-state`, and should be listed with the other fields at that level.

Reply:
Good catch!

Reply:
Sorry, I'm having trouble describing what I mean. This is what I was trying to get across: https://github.com/speth/cantera/commit/b7ec97097681613b9a85894ef683187696dc9064.

Reply:
Ok. When I cherry-pick your commit, the only difference from I see is that your version skips 

```
``coverage-dependencies``
    Mapping where keys are the name of species whose coverage affects
    thermodynamic properties of the node-owner species. The map values are
    the dependency entries including ``model`` and model-specific parameters
```

Is this the intent?

Reply:
It _should_ be that my branch moves that block up to the top, between the descriptions of the `transport` and `sites` fields. I'll just go ahead and push here.